tests/utils.js

```diff
@@ -21,6 +21,12 @@ export function getCss(style) {
       .join('')
 }
 
+export function getCssFromSheet(sheet) {
+  return [...sheet.renderer.getRules()]
```

Comment:
there is no `Array.from` in old browser, so some tests fall.
should we use polyfills for tests/`test env` or rewrite it with for?